resources/ory/profile-evaluation.yaml

```
@@ -5,24 +5,32 @@ global:
       persistence:
         enabled: false
         postgresql:
-          enabled: false
+          enabled: true
```

I think we should also keep false since this is also an azure-lite override.
```suggestion
          enabled: false
```